Content.Shared/Movement/Components/SpeedModifierContactsComponent.cs

```diff
@@ -13,10 +13,14 @@ public sealed partial class SpeedModifierContactsComponent : Component
     [AutoNetworkedField]
     public float WalkSpeedModifier = 1.0f;
 
-    [AutoNetworkedField]
     [DataField("sprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
     public float SprintSpeedModifier = 1.0f;
 
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField]
```
`DataField` includes `ViewVariables(VVAccess.ReadWrite)` nowadays. Old files still do it manually, but it is not needed anymore.
Also add documentation for the datafield. I know the other ones aren't documented either, but that does not mean that you cannot do better :)

Reply:
Thanks for the info, I've made the suggested change. Also, I took the liberty of cleaning things up and adding in the missing documentation throughout, I hope that's okay!